CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# v0.900.0
+This is a major overhaul that brings signficant performance improvements.
+New features:
+- Added "go to definition" for `templateUrl` and `styleUrls`.
+- Hover tooltip now shows the `NgModule` a directive is declared in.
+- Added `angular.ngdk` config for specifying location of `@angular/language-service`.
+- Added vscode command to restart the extension.
+- Display loading indicator while project is loading.
```

Comment:
Is the workspace/bundled/user-specified `@angular/language-service` a new change in this version?